.ci/validate_notebooks.py

```diff
@@ -78,16 +78,24 @@ def clean_test_artefacts(before_test_files, after_test_files):
             shutil.rmtree(file_path, ignore_errors=True)
 
 
-def run_test(notebook_path, report_dir, collect_reports, root):
-    print(f'RUN {notebook_path.relative_to(root)}')
-    report_file = report_dir / f'{notebook_path.name}_report.xml'
+def run_test(notebook_path, root):
+    print(f'RUN {notebook_path.relative_to(root)}', flush=True)
+    
     with cd(notebook_path):
-        existing_files = sorted(list(notebook_path.rglob("*")))
```

Comment:
Previously, the `existing_files` was always empty because of the `cd` and still relative `notebook_path`: there are no notebook folder in the notebook folder.